web/beacon-app/src/hooks/useResetPassword.ts

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
```

Comment:
There's a `useResetPassword` hook in `features/auth`. Should this one be deleted?

Reply:
I've mentioned this to you in Slack. I guess you don't see it. I retained it as I plan to use it soon.